### ISSUE TITLE
If 0 is used for parameters, no value is stored in <foreach> tag

### ISSUE THREAD

**Issue by yurseria** [OldBlackJoe/mybatis-mapper#5]
I put array in `parameters` with 0 value.

**code.js**
```javascript
var parameters = {
  IDs: [0, 1, 2, 3]
};
mybatisMapper.getStatement(Namespace, SqlID, Parameters, format)
```
and use `<foreach>` in xml.

**sql.xml**
```xml
<mapper namespace="test">
  <select id="test_1">
    select
    from
      user
    where
    in
    (
      <foreach collection="IDs" item="ID" separator=",">
      #{ID}
      </foreach>
    )
  </select>
</mapper>
```

but no ID is stored in `<foreach>`.

**Result**
```SQL
select
from
  user
where
in
(



)
```

**@mybatis-mapper/lib/convert.js, line 138~147**

```javascript
var convertForeach = function (children, param, namespace, myBatisMapper) {
  try{
    var collection = eval('param.' + children.attrs.collection);
    var item = children.attrs.item;
    var open = (children.attrs.open == null)? '' : children.attrs.open;
    var close = (children.attrs.close == null)? '' : children.attrs.close;
    var separator = (children.attrs.separator == null)? '' : children.attrs.separator;
    
    var foreachTexts = [];
    for (var j=0, coll; coll=collection[j]; j++){
```

**line 140** : `collection = [0, 1, 2, 3]` is assigned.
but in javascript, `collection[0] = 0` is **false**.
so for-loop are failed in **line 147**, and no value is stored `<foreach>`.

This means that if the parameter `array contains 0`, the index in the for statement will be `false` and the loop will be terminated.

**Comment by OldBlackJoe** [comment]
Thank you for your issue. I'll be patch this problem as soon as possible :-)

**Comment by OldBlackJoe** [comment]
Fixed in version 0.6.2
